pkg/talos/talos_image_factory_urls_data_source_test.go

```diff
@@ -234,3 +273,36 @@ data "talos_image_factory_urls" "this" {
 }
 `
 }
+
+func testAccTalosImageFactoryURLsInvalidVersionConfig() string {
+	return `
+provider "talos" {}
+
+data "talos_image_factory_urls" "this" {
+	talos_version = "invalid_version"
+	schematic_id = ""376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba""
+}
+`
+}
+
+func testAccTalosImageFactoryURLsValidVersionWithVConfig() string {
+	return `
+provider "talos" {}
+
+data "talos_image_factory_urls" "this" {
+	talos_version = "v1.7.5"
+	schematic_id = ""376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba""
+}
+`
+}
+
+func testAccTalosImageFactoryURLsValidVersionWithoutVConfig() string {
+	return `
+provider "talos" {}
+
+data "talos_image_factory_urls" "this" {
+	talos_version = "v1.7.5"
```

Comment:
I guess there should be no `v` prefix here

Reply:
You are correct, copy and paste error